marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-entities.xqy

```diff
@@ -105,6 +105,47 @@ declare function hent:wrap-duplicates(
   because of https://github.com/marklogic/entity-services/issues/359
 :)
 declare %private function hent:fix-options($nodes as node()*)
+{
```

Comment:
I recommend adding unit tests - either to generateSearchOptions.sjs (which I added to address DHFPROD-2580), or to new modules in the same directory as generateSearchOptions. I would just add them to that module - you can put each test into a separate function.

Reply:
Added the tests in the new commit